### DISCUSSION TITLE
DELETE /iocs/entities/indicators/v1

### DISCUSSION THREAD

**Discussion by Sylverd** [CrowdStrike/falconpy#1054] [Q&A] [answered]
Anyone can help me identify what values I need to add? I can't understand where the hash goes or how to add several hashes to delete, I don't know where to find the IDs either.

i using this code

```
from falconpy import APIHarnessV2

falcon = APIHarnessV2(client_id=CLIENT_ID,
 client_secret=CLIENT_SECRET
 )

id_list = 'ID1,ID2,ID3' # Can also pass a list here: ['ID1', 'ID2', 'ID3']

response = falcon.command("indicator_delete_v1",
 filter="string",
 from_parent=boolean,
 comment="string",
 ids=id_list
 )

print(response)
```

**Comment by crowdstrikedcs** [comment, marked as answer]
Hi @Sylverd thanks for the question!

When browsing the Falcon Console, you can view the "IOC Management" dashboard [here ](https://falcon.crowdstrike.com/iocs/) each of these records has a unique "ID" that will be used in your `id_list` above.

To get these values over API I recommend using the `indicator_combined` operation found [here](https://falconpy.io/Service-Collections/IOC.html#indicator_combined_v1). This operation allows you to query and view details for indicators in your account.You may additionally for those indicators of a specified type or value. For Example here is a sample response:
![image](https://github.com/CrowdStrike/falconpy/assets/43282232/5039869b-e158-4eba-aa85-94f54cf42ea9)
 
You can see this indicator has a `type` and `value` (cnn.com & domain) as well as an `id` which will be used to delete from your operation above. Here is an example: 

```
id_list = '0309d21b61b2682893a85d15394731a068d87ec221d73d8ea47a9b6d38565798'
response = falcon.command("indicator_delete_v1", ids=id_list)

print(response)
```

Try this out and let us know if you have any additional questions.

**Reply by Sylverd** [reply]
Hi @crowdstrikedcs

Thank you so much for the support it was very helpful

Regards!
